ci/before_deploy.sh

```diff
@@ -3,9 +3,9 @@
 curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
 chmod +x ./kubectl
 sudo mv ./kubectl /usr/local/bin/kubectl
-echo -n ${CI_SERVICE_ACCOUNT_KEY} | base64 -d > travis-service-account.json
-gcloud auth activate-service-account ${CI_SERVICE_ACCOUNT} --key-file travis-service-account.json --project=${PROJECT_ID}
+echo -n "${CI_SERVICE_ACCOUNT_KEY}" | base64 -d > travis-service-account.json
```

Comment:
If you talk about this line:
`gcloud auth activate-service-account ${CI_SERVICE_ACCOUNT} --key-file travis-service-account.json --project=${PROJECT_ID}` 
it is still here if you check the entire file.
I guess it's because i suppress a EOL blank space

Reply:
I was talking about the double quotes you added

Reply:
oh ! I think this come from you on an another project in fact if i remember well ^^
We had an issue with a non quoted one and we agreed that we always should do this for avoid issue like this.